docs/TheBook/src/main/markdown/cookbook-transport-security.md

```diff
@@ -45,7 +47,7 @@ Configuring Java
 
 dCache relies on Java for its transport security. Hardening Java is necessary to get the maximum security. If not configured properly, Java may support ciphers and protocols that are considered weak.
 
-To begin with, make sure you run Java 8 or newer. It's impossible to get an A rating in the Qualys SSL test without Java 8 or newer. Java 11 would be best because it adds support for TLSv1.3, but dCache has not been tested with Java 11 (as of October 2018).
+To begin with, make sure you run Java 8 or newer. It's impossible to get an A rating in the Qualys SSL test without Java 8 or newer. If you run dCache 5, you can use Java 11 which adds support for TLSv1.3.
```

Comment:
As above, the information is certainly very useful, but shouldn't reference a dCache version.

This patch is targeting `master` (==> dCache v5.1), so you can say simply (rephrasing, slightly)

    Using Java 11 will add support for TLSv1.3

(or similar)